opendcs-web-client/src/main/webapp/resources/js/datatables/datatables.js

```diff
@@ -14,16 +14,19 @@
  */
 
 class OpenDcsDataTable {
-  constructor(domId, properties, inlineOptions, actions, initialize) {
+  constructor(domId, properties, inlineOptions, actions, initialize, tableProperties) {
 	  console.log("In super constructor for OpenDcsDataTable.");
+      var defaultTableProperties = {
+                                      cellBorders: true
+                                  };
 	  //Remove # if the passed id starts with one.
 	  if (domId.startsWith("#"))
 	  {
 		  domId = domId.slice(1);
 	  }
     this.domId = domId;
     this.jq = $(`#${domId}`);
-    
+    this.tableProperties = tableProperties != null ? tableProperties : defaultTableProperties;
```

Comment:
i think needs needs an undefined check instead of or in addition to a null check.

Reply:
But it's been a few months since I've done javascript.

Reply:
Been a while, but I updated with an undefined check.  
Since the PR was already approved, I will just resolve and close the PR.